solvesCryptarithm/README

```diff
@@ -0,0 +1,12 @@
+solvesCryptarithm Autograder
+
+- /src has all the base files
+    - .py solution file
+    - driver.py which runs the test cases/autogrades and gives you a score
+    - student starter files with -handout at the end of the filename
+- running Makefile (make) will create...
+    - solvesCryptarithm-handout folder for students
+    - autograde.tar & solvesCryptarithm-handout.tar (copies of directories)
+- make test will reset test-autograder directory
+    - moves in autograde-Makefile as the new Makefile
+- running make in test-autograder will run the driver
```

Comment:
Can you add information about the assessment language, autograder language, environment packages (python 3), and assessment scenario? 

Reference https://github.com/autolab/autograders-examples/blob/main/python-twosumlab/README.md

Reply:
Since you require the python3 environment package, which isn't installed on the default autograding_image, you should also include a Dockerfile to create an image that has python. You can reference [Joey's](https://github.com/autolab/autograders-examples/blob/main/rust-mergesort/Dockerfile_rust) or [Damian's](https://github.com/autolab/autograders-examples/blob/main/Brainfuck-cat/Dockerfile_bf). Make sure python3 is included in the apt-get install